carAPI/src/main/resources/data.sql

```diff
@@ -1 +1 @@
-INSERT INTO car (brand, country, createdat, lastupdated, registration) VALUES ('Car1', 'Spain', '2021-03-23 11:54:40', '2021-03-23 11:54:40', '2021-03-23 11:54:40'), ('Car2', 'Spain', '2021-03-23 11:54:40', '2021-03-23 11:54:40', '2021-03-23 11:54:40');
\ No newline at end of file
+INSERT INTO car (id, brand, country, createdat, lastupdated, registration) VALUES ('f44be336-9dc0-11eb-a8b3-0242ac130003', 'Car1', 'Spain', '2021-03-23 11:54:40', '2021-03-23 11:54:40', '2021-03-23 11:54:40'), ('f44be548-9dc0-11eb-a8b3-0242ac130003', 'Car2', 'Spain', '2021-03-23 11:54:40', '2021-03-23 11:54:40', '2021-03-23 11:54:40');
```

Comment:
If id is main key, this query will only work once, later you will have to remove it from db again, maybe you can try the IF NOT EXISTS... postgresql query